src/main/java/appeng/client/texture/FlippableIcon.java

```diff
@@ -126,14 +128,39 @@ public IIcon getOriginal()
 
 	public void setFlip( final boolean u, final boolean v )
 	{
-		this.flip_u = u;
-		this.flip_v = v;
+		this.setFlipU( u );
+		this.setFlipV( v );
 	}
 
 	public int setFlip( final int orientation )
 	{
-		this.flip_u = ( orientation & 8 ) == 8;
-		this.flip_v = ( orientation & 16 ) == 16;
+		this.setFlipU( ( orientation & 8 ) == 8 );
+		this.setFlipV( ( orientation & 16 ) == 16 );
 		return orientation & 7;
 	}
+
+	public boolean isFlipU()
+	{
+		return this.flip_u;
+	}
+
+	public void setFlipU( final boolean flip_u )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)